packages/patternfly-4/react-core/src/components/Modal/Modal.tsx

```diff
@@ -17,6 +17,8 @@ export interface ModalProps extends React.HTMLProps<HTMLDivElement> {
   isOpen?: boolean;
   /** Complex header (more than just text), supersedes title for header content */
   header?: React.ReactNode,
+  /** document where the Modal will be created and shown (via React.Portal) */
+  targetDocument?: HTMLDocument;
```

Comment:
can we change this prop name to `appendTo` since that is a prop we already use in some components?
And then I would make the type more generic
`appendTo?: Element | ((ref: Element) => Element);`